packages/node_modules/@webex/internal-plugin-conversation/test/integration/spec/get.js

```diff
@@ -814,17 +814,72 @@ describe('plugin-conversation', function () {
           });
       });
 
-      it('should return searched-for activity with surrounding activities when jumpToActivity is called with an activity', () => {
-        const search = firstParentBatch[firstParentBatch.length - 1];
+      describe('jumpToActivity()', () => {
+        let _listActivitiesThreadOrderedSpy;
 
-        return jumpToActivity(search)
-          .then(({value}) => {
-            const searchedForActivityIx = findIndex(value, ['id', search.id]);
+        beforeEach(() => {
+          _listActivitiesThreadOrderedSpy = sinon.spy(webex.internal.conversation, '_listActivitiesThreadOrdered');
+        });
 
-            assert.isFalse(searchedForActivityIx === -1);
-            assert.isTrue(searchedForActivityIx > 0);
-            assert.isTrue(searchedForActivityIx < value.length);
-          });
+        afterEach(() => {
+          webex.internal.conversation._listActivitiesThreadOrdered.restore();
+        });
+
+
+        it('should return searched-for activity with surrounding activities when jumpToActivity is called with an activity', () => {
+          const search = firstParentBatch[firstParentBatch.length - 1];
+
+          return jumpToActivity(search)
+            .then(({value}) => {
+              const searchedForActivityIx = findIndex(value, ['id', search.id]);
+
+              assert.isFalse(searchedForActivityIx === -1);
+              assert.isTrue(searchedForActivityIx > 0);
+              assert.isTrue(searchedForActivityIx < value.length);
+            });
+        });
+
+        it('should return all activities in a space when jumping to an activity in a space with less activities than the asked-for activities limit', () => {
+          let conversation2, msg, actCount;
+
+          return webex.internal.conversation.create({participants: [scott.id]})
+            .then((c) => {
+              conversation2 = c;
+
+              return webex.internal.conversation.post(conversation2, {displayName: 'first message'});
+            })
+            .then((m) => {
+              msg = m;
+
+              return webex.internal.conversation.listActivities({conversationUrl: conversation.url});
+            })
+            .then((acts) => {
+              actCount = acts.length;
+
+              return jumpToActivity(msg);
+            })
+            .then(() => {
+              assert.isTrue(actCount < minActivities);
+            });
+        });
```

Comment:
```suggestion
        it('should return all activities in a space when jumping to an activity in a space with less activities than the asked-for activities limit', () => {
          return webex.internal.conversation.create({participants: [scott.id]})
            .then((c) => {
              return webex.internal.conversation.post(c, {displayName: 'first message'})
                .then((m) => {
                  return webex.internal.conversation.listActivities({conversationUrl: c.url})
                    .then((acts) => {
                      return jumpToActivity(m)
                        .then(() => acts.length);
                    })
                })
            })
            .then((actCount) => {
              assert.isTrue(actCount < minActivities);
            });  
        });
```

Reply:
You don't need to save off variables. You can pass them down or keep them in scope.